saleor/graphql/app/schema.py

```diff
@@ -63,17 +64,26 @@ class AppQueries(graphene.ObjectType):
         sort_by=AppSortingInput(description="Sort apps."),
         description="List of the apps.",
         permissions=[
+            AuthorizationFilters.AUTHENTICATED_STAFF_USER,
             AppPermission.MANAGE_APPS,
         ],
     )
-    app = graphene.Field(
+    app = PermissionsField(
         App,
         id=graphene.Argument(graphene.ID, description="ID of the app.", required=False),
         description=(
             "Look up an app by ID. If ID is not provided, return the currently "
-            "authenticated app. Requires one of the following permissions: "
-            f"{AuthorizationFilters.OWNER.name}, {AppPermission.MANAGE_APPS.name}."
+            "authenticated app. \n\nRequires one of the following permissions: "
+            f"{AuthorizationFilters.AUTHENTICATED_STAFF_USER.name} "
```

Comment:
Before only staff users with MANAGE_APPS could use this field, now we will allow to any staff that is authenticated, is that correct?

Reply:
@maarcingebala Extra space before the `\n` 😄

Reply:
@maarcingebala Yes, it is expected. We want to allow using the apps by any staff users. The JWT token that is passed to App's iframe will have a separate section with the staff user's permissions, so the app can decide what part of the logic should be available for the user.